### DISCUSSION TITLE
Encoders: jim vs ffmpeg

### DISCUSSION THREAD

**Discussion by fabiomazzi** [sorayuki/obs-multi-rtmp#353] [Q&A]
What's the difference between the jim and the ffmpeg versions of the encoders? For example:
NVIDIA NVENC H.264 [jim_nvenc]
and
NVIDIA NVENC H.264 (FFmpeg) [ffmpeg_nvenc]

**Comment by sorayuki** [comment]
The jim one uses texture as input type and the ffmpeg one use memory block as input type. If the previous one works well, use it.
________________________________
From: fabiomazzi ***@***.***>
Sent: Saturday, October 28, 2023 9:56:26 AM
To: sorayuki/obs-multi-rtmp ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [sorayuki/obs-multi-rtmp] Encoders: jim vs ffmpeg (Discussion #353)


What's the difference between the jim and the ffmpeg versions of the encoders? For example:
NVIDIA NVENC H.264 [jim_nvenc]
and
NVIDIA NVENC H.264 (FFmpeg) [ffmpeg_nvenc]

―
